pkgs/applications/graphics/tesseract/tesseract4.nix

```diff
@@ -12,6 +12,17 @@ stdenv.mkDerivation rec {
     sha256 = "1ca27zbjpx35nxh9fha410z3jskwyj06i5hqiqdc08s2d7kdivwn";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/tesseract-ocr/tesseract/commit/dbc79b09d195490dfa3f7d338eadac07ad6683f7.patch";
+      sha256 = "sha256-lGlg0etuU4RXfdq1QH2bYObdeGrFHKf9O8zMUAbfNIQ=";
+    })
```

Comment:
This patch is not necessary for tesseract4 to build. Other than that, it works perfectly (with and without this first patch).